mowgli_etl/pipeline/wdc/wdc_constants.py

```diff
@@ -6,4 +6,9 @@
 WDC_ARCHIVE_PATH = DATA_DIR / 'wdc' / 'extracted'
 WDC_CSV_FILE_KEY = 'wdc_csv_file'
 
+WDC_RE_DIMENSION_DECIMAL_STR = r"\d+\s\d+[hwl]"
```

Comment:
These regexes don't really belong here, since they're not shared between multiple modules? If it's only on one module, make it a private class constant:
```
class Whatever:
   __CONSTANT = "..."
  
  def f(self): self.__CONSTANT
```